aws/resource_aws_organizations_organization.go

```diff
@@ -106,7 +142,18 @@ func resourceAwsOrganizationsOrganizationRead(d *schema.ResourceData, meta inter
 		return fmt.Errorf("error describing Organization: %s", err)
 	}
 
+	log.Printf("[INFO] Listing Roots for Organization: %s", d.Id())
+	var roots []*organizations.Root
+	err = conn.ListRootsPages(&organizations.ListRootsInput{}, func(page *organizations.ListRootsOutput, lastPage bool) bool {
+		roots = append(roots, page.Roots...)
+		return !lastPage
+	})
+	if err != nil {
+		return err
```

Comment:
To provide context for operators and code maintainers, we should include some additional error messaging here, e.g.

```suggestion
		return fmt.Errorf("error listing AWS Organization (%s) roots: %s", d.Id(), err)
```